src/ray/gcs/redis_gcs_client.h

```diff
@@ -75,8 +76,10 @@ class RAY_EXPORT RedisGcsClient : public GcsClientInterface {
   /// one event loop should be attached at a time.
   Status Attach(boost::asio::io_service &io_service);
 
-  /// Use method Actors() instead
+  /// Use method Actors() instead.
   ActorTable &actor_table();
+  /// Use method Jobs() instead.
```

Comment:
what does this comment mean? Is it a TODO?

Reply:
Not a TODO. Prompt everyone not to use the old interface.